src/OrbitGl/GraphTrack.cpp

```diff
@@ -29,9 +29,6 @@ GraphTrack<Dimension>::GraphTrack(CaptureViewElement* parent, TimeGraph* time_gr
       series_(MultivariateTimeSeries<Dimension>(series_names)) {
   SetName(name);
   SetLabel(name);
-
-  collapse_toggle_->SetState(TriangleToggle::State::kExpanded,
-                             TriangleToggle::InitialStateUpdate::kReplaceInitialState);
```

Comment:
Thanks for making this change. May I know what is the default state of `GraphTrack` now? Is it expanded by default?

Reply:
I've split the state into collapsed/expanded and collapsable/not-collapsable. The default is expanded and collapsable.